docs/software/sciapps/gromacs.md

```diff
@@ -1,4 +1,161 @@
+
+5. **User-Friendly Interface**: GROMACS provides a command-line interface (CLI) and a set of well-documented input and control files, making it accessible to both novice and expert users. It offers flexibility in defining system parameters, simulation conditions, and analysis options through easily modifiable input files.
+
+6. **Integration with Other Software**: GROMACS can be integrated with other software packages and tools to perform advanced analysis and extend its capabilities. It supports interoperability with visualization tools like VMD and PyMOL, analysis packages like GROMACS Analysis Tools (GROMACS Tools) and MDAnalysis, and scripting languages such as Python, allowing users to leverage a wide range of complementary tools.
+
+## Licensing Terms & Conditions
+
+GROMACS is a joint effort, with contributions from developers around the world: users agree to acknowledge use of GROMACS in any reports or publications of results obtained with the Software (see [GROMACS Homepage](https://www.gromacs.org/about.html) for details).
+
+## Daint on Alps (GH200)
+
+### Setup
+On ALPS, we provide pre-built user environments containing GROMACS alongside all the required dependencies for the GH200 hardware setup. To access the `gmx_mpi` executable, we do the following:
+
+```bash
+uenv image find 							  # list available images
+
+uenv image pull gromacs/VERSION:TAG.          # copy version:tag from the list above
+uenv start gromacs/VERSION:TAG --view=gromacs # load the gromacs view
+
+gmx_mpi --version                             # check GROMACS version
+```
+
+The images also provide two alternative views, namely `plumed` and `develop`.
+After starting the pulled image using `uenv start ...` , one may do the following see the available views.
+
+```bash
+$ uenv status
+/user-environment:gromacs-gh200
+  GPU-optimised GROMACS with and without PLUMED, and the toolchain to build your own GROMACS.
+  modules: no modules available
+  views:
+    develop
+    gromacs
+    plumed
+```
+
+The `develop` view has all the required dependencies or GROMACS without the program itself. This is meant for those users who want to use a customized variant of GROMACS for their simulation which they build from source. This view makes it convenient for users as it provides the required compilers (GCC 12) along with the dependencies such as CMake, CUDA, hwloc, Cray MPICH, among many others which their GROMACS can use during build and installation. Users must enable this view each time they want to use their **custom GROMACS installation**.
+
+The `plumed` view contains GROMACS 2022.5 (older version) with PLUMED 2.9.0. This is due to the compatibility requirements of PLUMED. CSCS will periodically update these user environment images to feature newer versions as they are made available.
```

Comment:
This is very specific to the current image. IIRC, the new image will have the same version of GROMACS in both views. Is there a way to make it more general to avoid going out of date?